src/Core/src/Handlers/WebView/WebViewHandler.Android.cs

```diff
@@ -37,10 +50,23 @@ public override void SetVirtualView(IView view)
 		protected override void DisconnectHandler(AWebView platformView)
 		{
 			platformView.StopLoading();
+			
+			_webViewClient?.Dispose();
+			_webChromeClient?.Dispose();
 
 			base.DisconnectHandler(platformView);
 		}
 
+		protected virtual WebViewClient GetWebViewClient()
```

Comment:
why are we adding this if we have mappers?

Reply:
Right, is not necessary. I removed it.